### DISCUSSION TITLE
flink stream 1s导入一次，偶发be崩溃

### DISCUSSION THREAD

**Discussion by zyw8136** [apache/doris#7476] [A - General / Q&A]
be版本：0.15

崩溃日志

#0  0x00002abd22d25207 in raise () from /lib64/libc.so.6
#1  0x00002abd22d268f8 in abort () from /lib64/libc.so.6
#2  0x000000000125a973 in google::DumpStackTraceAndExit () at src/utilities.cc:160
#3  0x00000000030131dd in google::LogMessage::Fail () at src/logging.cc:1650
#4  0x0000000003015319 in google::LogMessage::SendToLog() () at src/logging.cc:1607
#5  0x0000000003012d59 in google::LogMessage::Flush() () at src/logging.cc:1476
#6  0x0000000003015909 in google::LogMessageFatal::~LogMessageFatal (this=this@entry=0x7ffe30358fe0, __in_chrg=<optimized out>) at src/logging.cc:2226
#7  0x0000000001989606 in doris::ThreadPool::~ThreadPool() () at /opt/rh/devtoolset-10/root/usr/include/c++/10/ext/new_allocator.h:89
#8  0x000000000161c947 in operator() (__ptr=0x6ec4e00, this=0x6dc8b50) at /opt/rh/devtoolset-10/root/usr/include/c++/10/bits/unique_ptr.h:79
#9  ~unique_ptr (this=0x6dc8b50, __in_chrg=<optimized out>) at /opt/rh/devtoolset-10/root/usr/include/c++/10/bits/unique_ptr.h:361
#10 ~MemTableFlushExecutor (this=0x6dc8b50, __in_chrg=<optimized out>) at ../src/olap/memtable_flush_executor.h:95
#11 operator() (this=0x735e2c0, __ptr=0x6dc8b50) at /opt/rh/devtoolset-10/root/usr/include/c++/10/bits/unique_ptr.h:85
#12 operator() (__ptr=0x6dc8b50, this=0x735e2c0) at /opt/rh/devtoolset-10/root/usr/include/c++/10/bits/unique_ptr.h:79
#13 ~unique_ptr (this=0x735e2c0, __in_chrg=<optimized out>) at /opt/rh/devtoolset-10/root/usr/include/c++/10/bits/unique_ptr.h:361
#14 doris::StorageEngine::~StorageEngine (this=0x735e000, __in_chrg=<optimized out>) at ../src/olap/storage_engine.cpp:142
#15 0x0000000001337a97 in main () at ../src/service/doris_main.cpp:282
#16 0x00002abd22d113d5 in __libc_start_main () from /lib64/libc.so.6
#17 0x000000000153170e in _start () at /opt/rh/devtoolset-10/root/usr/include/c++/10/bits/char_traits.h:364

**Comment by morningman** [comment]
You need to see be.INFO, there may be fatal log start with "F"
